tests/system/action/meeting/test_delete.py

```diff
@@ -429,6 +430,47 @@ def test_delete_with_locked_meeting(self) -> None:
             lock_meeting=True,
         )
 
+    def test_delete_with_locked_meeting_orgaadmin(self) -> None:
+        self.base_permission_test(
+            {},
+            "meeting.delete",
+            {"id": 1},
+            OrganizationManagementLevel.CAN_MANAGE_ORGANIZATION,
+            True,
+            lock_meeting=True,
```

Comment:
You could use a keyword for both boolean arguments.

Reply:
Done

Reply:
I think for people who are new to the code, it's great to to be able to guess the arguments function by its name at first glance. Also this looks like you are leaving out arguments in between which you are not. But I'm also doing some bean counting here.